CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+# 107.0.0
+
+In this major release we deleted the deprecated themer classes: `MDCAlertScheme` and `MDCAlertControllerThemer`.
+
+## Breaking changes
+
+Deleted `MDCAlertScheme` and `MDCAlertControllerThemer` class, please use `MDCAlertController (MaterialTheming)` extension instead.
+
+## New features
+
+The new delegate MDCAlertControllerDelegate protocol exposes dialogs life cycle events that are useful for impression logging."
```

Comment:
Should Dialogs be capitalized? Also, it looks like there's a closing quotation mark at the end of the sentence with no opening quotation mark before it.

Reply:
>The new MDCAlertControllerDelegate protocol exposes Dialogs life cycle events that are useful for impression logging.

Reply:
This has been deleted.